src/api.ts

```diff
@@ -20,6 +20,8 @@ router.get('/clear/:type/:id', async (req, res) => {
 
 router.get('/:type/:id', async (req, res) => {
   const { type, id } = req.params;
+  const { fb } = req.query;
+
   const { address, network, w, h } = await parseQuery(id, type, req.query);
   const key1 = sha256(
     JSON.stringify({ type, network, address, w: constants.max, h: constants.max })
```

Comment:
Awesome! It seem to work well. Just here we should add the fb param with 'blockie' or 'jazzicon' to create the key1 and also key2, these keys are used to get and store caches, if we dont include them here we may render an image with jazzicon as fallback even if the request doesnt include fb param.